Working branch

qusai122 opened the pull request:
On this project:

- I have successfully configured the HTML & CSS Linters.
- I have added a Readme file.
- I have added Html and CSS files with content in it!
- I have used Lenters for HTML & CSS.
- I have also added .gitignore file and mentioned unwanted files to push( /node_modules & Test.md).

Commit:
create HTML file

Commit:
create CSS file

Commit:
Set up a linter for HTML and CSS

Commit:
descriptive README file

Commit:
adding test.md and ignore it

mahbubul14 commented on README.md:
```diff
@@ -1 +1,185 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
```
- [ ] Kindly describe your project in 1 or 2 sentences. :+1:

mahbubul14 commented on README.md:
```diff
@@ -1 +1,185 @@
+  - [Answer_1]
+- **[Question_2]**
+  - [Answer_2]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [ ] Please include an MIT license file to your project. The link to your license should be included in your README file. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project. :+1:

qusai122 replied:
hello thanks for the good response,
the changes are ready now. :)

qusai122 replied:
should I request anew review now?

qusai122 replied:
@mahbubul14

mahbubul14 reviewed (changes requested):
Hi @qusai122,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixing bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Commit:
adding MIT License

Commit:
including text to the MIT License

AlphaNtihinduka commented on README.md:
```diff
@@ -1 +1,188 @@
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
- [ ] As the previous reviewer suggested, the link to your license should be included in your README file and active. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project. 👍

qusai122 replied:
@AlphaNtihinduka 
thanks for the reply
it's done now, but I made the edit directly to the main branch.
this was my last review request what should I do now?

AlphaNtihinduka replied:
> @AlphaNtihinduka thanks for the reply it's done now, but I made the edit directly to the main branch. this was my last review request what should I do now?

Dear @qusai122 you should ensure that all changes are applied to the `feature-branch` instead of `main`. You can request for the second opinion. Once offered the chance you can ask for the rereview in the dashboard.

AlphaNtihinduka reviewed (changes requested):
## Hi @qusai122 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- README file is almost professional :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

## Required changes :recycle: 
Please consider the inline comments.

## Optional suggestions
- [ ] [OPTIONAL]  The branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). I therefore suggest to checkout to new branch and use the correct name and mention the old PR in the new PR (This can be done by attaching the old PR link in the new PR). This therefore implies you have to request for a review using new PR instead of this old one ☺️ .

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

frank1738 commented on README.md:
```diff
@@ -1 +1,188 @@
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
- [ ] The link to your license is still broken. Kindly adjust the paths to look like  `[MIT](LICENSE.md)` in your readme file so that it does not raise a `404` page error

frank1738 reviewed (changes requested):
Hi @qusai122 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/hXz6GN0ZilLwhgLo49/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Fix LICENSE link

frank1738 reviewed (approved):
Hi @qusai122 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media3.giphy.com/media/J8FZIm9VoBU6Q/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Pull Request summary 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

qusai122 commented:
On this project:

I have successfully configured the HTML & CSS Linters.
I have added a Readme file.
I have added Html and CSS files with content in it!
I have used Lenters for HTML & CSS.
I have also added .gitignore file and mentioned unwanted files to push( /node_modules & Test.md).
